app.py

```diff
@@ -316,9 +339,17 @@ def cli(
             )
             _LOGGER.info(f"Total number of solver workflows scheduled for DS packages: {scheduled_solvers}!")
             total_scheduled_solvers += scheduled_solvers
-
         elif dry_run:
             _LOGGER.info("dry-run: not scheduling data science packages solver jobs!")
+    else:
+        if data_science_packages_offset is not None:
+            _LOGGER.warning(
+                "Provided --data-science-packages-offset but solving data science packages will not be scheduled"
+            )
+        if data_science_packages_count is not None:
+            _LOGGER.warning(
+                "Provided --data-science-packages-offset but solving data science packages will not be scheduled"
+            )
```

Comment:
is this required, if the solve_data_science_package, is not set then, irrespective of the offset and count, packages will not be scheduled to solve, I guess one warning would be enough. right?